tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "## You are using the Python ARM Radar Toolkit (Py-ART), an open source\n",
+      "## library for working with weather radar data. Py-ART is partly\n",
+      "## supported by the U.S. Department of Energy as part of the Atmospheric\n",
+      "## Radiation Measurement (ARM) Climate Research Facility, an Office of\n",
+      "## Science user facility.\n",
+      "##\n",
+      "## If you use this software to prepare a publication, please cite:\n",
+      "##\n",
+      "##     JJ Helmus and SM Collis, JORS 2016, doi: 10.5334/jors.119\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pyart\n",
+    "import xarray as xr\n",
+    "import s3fs\n",
+    "import numpy as np\n",
+    "import datetime\n",
+    "import matplotlib.pyplot as plt\n",
+    "import cartopy.crs as ccrs\n",
+    "from matplotlib.pyplot import cm as cmaps\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# read in radar data\n",
+    "radar = pyart.io.read_nexrad_archive(\"s3://noaa-nexrad-level2/2021/05/26/KGLD/KGLD20210526_155623_V06\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# read in satellite data\n",
+    "fs = s3fs.S3FileSystem(anon=True)\n",
+    "aws_url = 's3://noaa-goes16/ABI-L2-MCMIPC/2021/146/15/OR_ABI-L2-MCMIPC-M6_G16_s20211461556154_e20211461558539_c20211461559030.nc'\n",
+    "\n",
+    "goes_data = xr.open_dataset(fs.open(aws_url), engine='h5netcdf')\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''\n",
+    "Because the GOES data comes in without latitude/longitude values, we need to calculate those. \n",
+    "'''\n",
+    "\n",
+    "def lat_lon_reproj(g16nc):\n",
```

Comment:
This can also be done using the Proj library, I'll check the results and see if this gives equal values

Reply:
Checked against reprojection using pyproj and the WGS84 ellipsoid and the results are near identical, so this looks good.

Here is the code I use for comparison:
https://github.com/w-k-jones/tobac-flow/blob/55b6a18abdefdfb4f3434cfc6d9974dc2694715b/src/tobac_flow/abi.py#L7-L29